internal/core/items/component_container.rs

```diff
@@ -163,6 +164,7 @@ impl Item for ComponentContainer {
             panic!("Internal compiler error: ComponentContainer had no child.");
         };
 
+        // This has two children: first the background rect and then the dynamic node
```

Comment:
I don't understand this comment. Is that outdated? I think there is anyone child, and the rectangle is not a children